### PR TITLE
Add new newsletter intro content

### LINKED ISSUE #69387: Newsletter Intro (design feedback)
Apply design feedback for newsletter intro page

HSljkKnYECEEegL3DqP2xC-fi-6827%3A68730

Context: pdDR7T-iV-p2#comment-270

### PR DESCRIPTION
**Pull request by heavyweight**
#### Proposed Changes
| Before  | After |
| ------------- | ------------- |
| Screenshot 2022-10-24 at 16 54 36 |  Screenshot 2022-10-24 at 16 53 53 |

- We need to update the newsletter intro page according to the new design
- We provide a mechanism to display the jetpack logo in the header
- In case the user locale is English or we have the new translations ready we display the new newsletter intro page.
 
#### Testing Instructions
- To test the new intro page visit the newsletter intro page with english locale. Test with different viewport sizes and compare the designs
- Check the link in bio flow for possible regression
- Check the newsletter flow with locale different than english for possible regressions

Related to #69387

### CODE REVIEW

**Review by alshakero — CHANGES_REQUESTED** [review]
It seems we'll have to go back to `contain` (vs cover) on mobile. The ripple effect of the background is lost on mobile.

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/7595429

Thank you @heavyweight for including a screenshot in the description! This is really helpful for our translators.

File: client/landing/stepper/declarative-flow/internals/steps-repository/intro/styles.scss [hunk 1]
```diff
@@ -4,44 +4,69 @@
...
-		background-size: auto 30%, auto 50%;
-
-
-		@include break-huge {
-			background-size: auto 55%, auto 80%;
-			background-position-y: 100%, 136px;
+
+		.step-container {
+			&.is-using-new-intro-design {
+				background-image: url(calypso/assets/images/onboarding/newsletter-intro-bottom.png);
```
Comment (alshakero) [inline client/landing/stepper/declarative-flow/internals/steps-repository/intro/styles.scss]: This bloody image is 971KB, would you be able to make a smaller one and use it here, and use this big one only on desktop?

**Review by alshakero — APPROVED** [review]
Works perfectly. Left one note.

File: client/landing/stepper/declarative-flow/internals/steps-repository/intro/styles.scss (hunk 1 above)
Comment (heavyweight) [reply client/landing/stepper/declarative-flow/internals/steps-repository/intro/styles.scss]: For sure 👍
Makes sense to me

**Comment by heavyweight** [comment]
@alshakero pushed a smaller image for mobile
I can compress the one for desktop up to `400kb` but it doesn't look good imo.
Set it to draft to wait for the translators. Should be ready on Monday.
That way we merge code without `hasTranslation` checks.

**Review by alshakero — APPROVED** [review]
Ship ship 🚢

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/7595429).

**Pull request merged**